internal/node/node.go

```diff
@@ -253,15 +253,21 @@ func NewBus(cfg BusConfig, dir string, seed types.PrivateKey, l *zap.Logger) (ht
 		dbConn = stores.NewSQLiteConnection(filepath.Join(dbDir, "db.sqlite"))
 	}
 
-	hooksMgr := webhooks.NewManager(l.Named("webhooks").Sugar())
-	alertsMgr := alerts.NewManager(hooksMgr)
+	alertsMgr := alerts.NewManager()
```

Comment:
Would it be slightly cleaner to pass a pointer to a broadcaster in the constructor and default to the NoOp broadcaster if it's nil? That avoids `RegisterWebhookBroadcaster` and the `panic` we have to throw.

Reply:
We can't do that unfortunately. Because of the dependencies the database, alert manager and broadcaster have.